Source/Core/Common/HttpRequest.cpp

```diff
@@ -221,8 +221,8 @@ HttpRequest::Response HttpRequest::Impl::Fetch(const std::string& url, Method me
   curl_easy_getinfo(m_curl.get(), CURLINFO_RESPONSE_CODE, &response_code);
   if (response_code != 200)
   {
-    ERROR_LOG(COMMON, "Failed to %s %s: server replied with code %li and body\n\x1b[0m%.*s", type,
-              url.c_str(), response_code, static_cast<int>(buffer.size()), buffer.data());
+    ERROR_LOG_FMT(COMMON, "Failed to {} {}: server replied with code {} and body\n\x1b[0m{:.{}}",
```

Comment:
Just wondering, why does this one have an ANSI reset code in it?

Reply:
I legitimately don't know

Reply:
For the record:

```
[23:44:56] <@Bh44L> leoetlino: do you remember why that log message in 9187 has an ANSI reset code in it?
[23:45:16] <@Bh44L> came from 5599
[23:47:05] <@leoetlino> uh, I think it was to make the response body stand out more in stderr output
[23:48:01] <@leoetlino> it's not strictly required, but it makes it easier to see where the body begins and ends when you have several ERROR_LOGs in a row
[23:56:40] <@Bh44L> oh, right, log goes to console as well
[23:56:42] <@Bh44L> ¯\_(ツ)_/¯
```